app/src/ui/app.tsx

```diff
@@ -149,6 +151,10 @@ export class App extends React.Component<IAppProps, IAppState> {
         },
         { timeout: ReadyDelay }
       )
+
+      window.setInterval(() => {
```

Comment:
Is there any need to clean up this resource or can the GC infer when a timer is no longer needed?

Reply:
The handle is tied to the lifecycle of `App`, so unless we were somehow recreating this interval and needed to `clearInterval` the previous value I wouldn't worry about it.

Reply:
This will keep running forever, and its existence will stop the instance from getting garbage collected. I’d recommend cancelling it when the component unmounts.